docs/enriching-your-data/available-enrichments/index.md

```diff
@@ -23,6 +23,7 @@ Currently, it’s not possible to configure more than one instance of the same e
 | [Currency Conversion](/docs/enriching-your-data/available-enrichments/currency-conversion-enrichment/index.md)     | Convert the values of all transactions to a specified base currency using Open Exchange Rates API. *(Please note the [limitations](/docs/enriching-your-data/available-enrichments/currency-conversion-enrichment/index.md).)* |
 | [Referer Parser](/docs/enriching-your-data/available-enrichments/referrer-parser-enrichment/index.md)              | Extracts attribution data from referer URLs.                                                                                                                                                                                   |
 | [Campaign Attribution](/docs/enriching-your-data/available-enrichments/campaign-attribution-enrichment/index.md)   | Choose which querystring parameters will be used to generate the marketing campaign fields. *If you do not enable the campaign_attribution enrichment, those fields will not be populated.*                                    |
+| [Cross Navigation Enrichment](/docs/enriching-your-data/available-enrichments/cross-navigation-enrichment/index.md)   | Parse the extended cross navigation format in _sp querystring parameter and attach the cross_navigation context to an event.                                                                                                                                        |
```

Comment:
Could you state here that it’s new as of Enrich 4.1.0? Also, could you update `docs/reusable/enrichment-list/_index.md` the same way?

Reply:
I've updated `docs/reusable/enrichment-list/_index.md` but couldn't find any place it is used. Should we remove it ?

Reply:
Oh! Yes, then we should remove, sorry!